python/datashaper/datashaper/engine/verbs/verbs_mapping.py

```diff
@@ -11,35 +11,62 @@
 from datashaper.table_store import TableContainer
 
 
-def verb(*args, **kwargs) -> Callable:
+def verb(name: str, does_not_mutate_input_tables: bool = False, **kwargs) -> Callable:

```

Comment:
Generally one should avoid using negative variable names for booleans. I see you follow the guideline avoid using variables names that enable setting "false" by default.
What about a simpler name, less prone to generating confusion? :) preserve_input_table, keep_input_clean?

Reply:
That's a fair point, I renamed this to `treats_input_tables_as_immutable`